src/Workspaces/Remote/ServiceHub/Services/RemoteHostService.cs

```diff
@@ -223,6 +226,31 @@ private static void SetGlobalContext(string serializedSession)
             FatalError.NonFatalHandler = WatsonReporter.Report;
         }
 
+        private static void EnsureCulture(int uiCultureLCID, int cultureLCID)
+        {
+            // this follows what VS does
+            // http://index/?leftProject=Microsoft.VisualStudio.Platform.AppDomainManager&leftSymbol=wok83tw8yxy7&file=VsAppDomainManager.cs&line=106
+            try
+            {
+                // set default culture for Roslyn OOP
+                CultureInfo.DefaultThreadCurrentUICulture = new CultureInfo(uiCultureLCID);
+                CultureInfo.DefaultThreadCurrentCulture = new CultureInfo(cultureLCID);
+            }
+            catch (Exception ex) when (ExpectedCultureIssue(ex))
+            {
+                // ignore expected culture issue
+            }
+        }
+
+        private static bool ExpectedCultureIssue(Exception ex)
+        {
+            // report exception
+            WatsonReporter.Report(ex);
+
+            // ignore expected exception
```

Comment:
If they are expected, why are we reporting them to Watson?

Reply:
I still want to see how often VS uses wrong LCID and what LCID it uses.